### DISCUSSION TITLE
cannot find ollama guide in docs

### DISCUSSION THREAD

**Discussion by morandalex** [Portkey-AI/gateway#247] [Q&A] [answered]
I cannot find the ollama guide
the following does not exist:
https://portkey.ai/docs/welcome/integration-guides/ollama

**Comment by vrushankportkey** [comment, marked as answer]
@morandalex Apologies! It seems this was missed. We had updated the Ollama guide sometime back - you can find it here - https://portkey.ai/docs/welcome/integration-guides/ollama

Hope it helps! And apologies for missing out on this thread 🙏
